Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM semtech/mu-javascript-template:1.3.2
+FROM semtech/mu-javascript-template:1.8.0
 LABEL maintainer=info@redpencil.io
 
 ENV CRON_PATTERN '0 5 1 * * *'
```

Comment:
Should we keep the cron pattern here? Or define it in the relevant app?

Putting in an app's docker compose file would make it easier to update, instead of having it as part of the service's Dockerfile.

Reply:
Both options works

But that's a good question, I'm not sure if we should keep it in the dockerfile, I mean we can always provide a default value in the service if the ENV is not provided in the docker-compose. At least, it's similar to other services.